com/course/zenkouv/src/Lecture3/Task1/Knight.java

```diff
@@ -37,13 +35,21 @@ public void addOutfit(Outfit outfit) {
         }
     }
 
+    public void removeOutfit(Outfit outfit) {
+        try {
+            outfits.remove(outfit);
+        } catch (IndexOutOfBoundsException ex) {
```

Comment:
you will never get `IndexOutOfBoundsException ` here
please read List.remove(T t) documentation

Reply:
fix